From 1ca50949291af5813412ae74596f36922aa415e9 Mon Sep 17 00:00:00 2001
From: tgeisler <tgeisler@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Mon, 4 Jun 2012 14:44:11 +0000
Subject: [PATCH 092/462] makefile.common.mak Buildscript failed if
 VC_COMPILER_VERSION was set to a vaulue different VC10 and not empty

git-svn-id: svn://localhost/SelfServiceBuild/trunk@123 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.common.mak | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SelfServiceBuild/Build/cpp/makefile.common.mak b/SelfServiceBuild/Build/cpp/makefile.common.mak
index 45af0e4..1d26556 100644
--- a/SelfServiceBuild/Build/cpp/makefile.common.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.common.mak
@@ -214,6 +214,7 @@ PP_SOAPGEN=..\..\$(_SOAPGEN)
 GRACEFULLCRASH   = -INCLUDE:_gracefullycrash $(_LIB)\gracefullycrash.lib
 STDDEFINES       = -DWIN32 -DWINNT $(TAO_DEFINES) -DMSYSTHREAD_USE_SEH
 
+
 !if "$(VC_COMPILER_VERSION)" == "VC10"
 !if "$(PLATFORM_DEBUG)" == "TRUE"
 STDCFLAGS        = -nologo -c -MDd -W3 -GR -EHsc -Zi -Od -wd4290 $(STDDEFINES) -I$(_INC) -I$(_GEN)
@@ -221,13 +222,12 @@ STDCFLAGS        = -nologo -c -MDd -W3 -GR -EHsc -Zi -Od -wd4290 $(STDDEFINES) -
 STDCFLAGS        = -nologo -c -MD  -W3 -GR -EHsc -Zi -Od -wd4290 $(STDDEFINES) -I$(_INC) -I$(_GEN)
 !endif
 STDDEFINES       = $(STDDEFINES) $(BOOST_DEFINES) -D_CRT_SECURE_NO_WARNINGS -D_CRT_NON_CONFORMING_SWPRINTFS -D_COMPILER_VERSION_VC10
-# to decide optimization level 
 MY_CFLAGS_1 	  = -GR -Gy 
 MY_CFLAGS_2      = -GR
 MY_CFLAGS_3      = -GR
 MY_CFLAGS_4      = -EHsc -GR
-!endif
-!if "$(VC_COMPILER_VERSION)" == ""
+!else
+# standard for VC6 if not defined as VC10
 STDCFLAGS        = -nologo -c -MD -W3 -GR -GX -Zi -Od $(STDDEFINES) -I$(_INC) -I$(_GEN)
 STDDEFINES       = $(STDDEFINES) -D_COMPILER_VERSION_VC6
 MY_CFLAGS_1 	  = -Gi -GR -G5 -Gy
-- 
2.41.0.windows.1

